src/pages/layout/Search/SearchInput.tsx

```diff
@@ -23,9 +35,14 @@ export default function SearchInput({...params}) {
             </InputAdornment>
           ),
         }}
-        placeholder="Search transactions"
-        helperText="Account Address / Txn Version & Hash / Block Height & Version"
+        placeholder="Search Explorer"
+        helperText="Account Address / Txn Hash or Version / Block Height or Version"
         fullWidth
+        onKeyPress={(event) => {
```

Comment:
I think it makes more sense to have it be in onSubmit on the form. This will work better if the form is submitted programmatically (i.e. it's more accessible).